src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -159,22 +159,22 @@ void KeyboardShortcuts::Set(int32_t key)
 
 int32_t KeyboardShortcuts::GetFromKey(int32_t key)
 {
-    for (int32_t i = 0; i < SHORTCUT_COUNT; i++)
+    for (size_t i = 0; i < GetShortcutsCount(); i++)
     {
         if (key == _keys[i])
         {
-            return i;
+            return static_cast<int32_t>(i);
         }
     }
-    return SHORTCUT_UNDEFINED;
+    return static_cast<int32_t>(Input::Shortcut::Undefined);
 }
 
 std::string KeyboardShortcuts::GetShortcutString(int32_t shortcut) const
 {
     utf8 buffer[256] = { 0 };
     utf8 formatBuffer[256] = { 0 };
     uint16_t shortcutKey = _keys[shortcut];
-    if (shortcutKey == SHORTCUT_UNDEFINED)
+    if (shortcutKey == static_cast<uint16_t>(Input::Shortcut::Undefined))
```

Comment:
this is not Input::Shortcut::Undefined but a different variable with the same value.